### DISCUSSION TITLE
.NET Core Minimal API - Invalid anti-forgery token

### DISCUSSION THREAD

**Discussion by RaphaelVidinhaElumini** [balta-io/forum#709] [Tira Dúvidas] [answered]
Fala pessoal, estou tendo um problema aqui na construção de uma API com .NET, utilizei como base o curso Fullstaclk .NET, porém, na hora de criar endpoints POST estou tomando este erro antes msm do endpoint ser executado:

`Microsoft.AspNetCore.Http.BadHttpRequestException: Invalid anti-forgery token found when reading parameter "IFormFile request" from the request body as form.`

![image](https://github.com/user-attachments/assets/c6916968-9f21-43a7-89b7-3fd8e275bb35)

Este endpoint é um post que faz upload de uma planilha em excel.
Alguém já passou por isso? sabe como reverter?

**Comment by malighieri** [comment, marked as answer]
Olá!
Acredito que a primeira resposta para a issue [Calling Minimal API with IFormFile gets "Invalid anti-forgery token"](https://github.com/dotnet/aspnetcore/issues/49929#issuecomment-1670156132) no Git do AspNetCore resolve seu problema.

**Reply by RaphaelVidinhaElumini** [reply]
Obrigado amigo, tão simples haha valeu msm!
